ledger-core/conveyor/smachine/main/example/catalogC.plantuml

```diff
@@ -0,0 +1,7 @@
+@startuml
+Init : INHERITED
+Init --> [*]
+Init --> State1 : JumpExt
+State1 : INHERITED
+State1 --> State1 : ThenRepeat
```

Comment:
Would be nice to show full predicate, e.g. Sleep().ThenRepeat()

Reply:
Agree, but that comment related to utility not in a generator. You can leave a comment in issue https://insolar.atlassian.net/browse/PLAT-442